config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
       "slug": "grains",
-      "topics": []
+      "topics": [
+        "Control-flow (if-else statements)",
+        "Bitwise operations",
+        "Mathematics",
+        "Floating-point numbers",
```

Comment:
It was less the use of floating-point numbers and more the type conversion to a float that occurs; `math.Pow(2, float64(n-1))`
Now I write this, I've just checked the topics list and see there is a Type conversion topic, so I think I'll swap it out for that, what do you think @petertseng @ferhatelmas ?